### DISCUSSION TITLE
Preview newsletter link on the newsletter post edit page

### DISCUSSION THREAD

**Discussion by X-Raym** [hizzle-co/noptin#577] [Ideas]
Hi,

We have access to a nice Preview feature on the newsletter list page, but preview isnt available in the edit page.
It would be convenient to have a button or a metabox with a link for that. This will prevent the need of going pack in newsletter list page.

Thx!

**Comment by picocodes** [comment]
It's not possible to add the preview link in a meta box since you need to save a campaign before you can preview it. I've added a preview link in the "campaign saved" success message.

**Comment by X-Raym** [comment]
@picocodes it could added inly after first save,
just like the wordpress Permalink / Url slug post metabox entry really.

In success message means it needs pressing save button and wait for page reload before accessing the preview.

**Comment by picocodes** [comment]
This is now implemented.

Thanks.

**Comment by X-Raym** [comment]
Perfect, thanks for your consideration 👍
